x-pack/plugins/observability/public/utils/slo/translations.ts

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
Ah, I was just moving all the translations in the other pages into their owners component files, and colocating them there. Would be nice for consistency if we could that here too.

Reply:
But they are used in more than one place

Reply:
Maybe `translations.ts` is not a great name though. But these functions:
1. Are used from different components / pages
2. Implement some logic in order to display the correct label (good to avoid duplication here)

Reply:
I've renamed it to `labels.ts`

Reply:
Maybe my comment is a bit late, but since we have `translations.ts` in other places, I think this name is more consistent than `labels.ts` :)

Reply:
I don't mind either way :)